docs/docs/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Setting Up and Running Jenkins
+---
+
+## Setting Up and Running Jenkins
+
+### Quickstart
+
+Jenkins on Mesos works by running the Jenkins master as a one-instance Marathon
+application. Once the Jenkins master comes online, it registers itself as
+a Mesos framework (using the [jenkins-mesos][jenkins-mesos-plugin] plugin).
+
+This project aims to make it easy to run Jenkins as a Marathon service, both
+on open source Marathon and on the Mesosphere DCOS, as described on the
+[Overview](../) page.
+
+### Requirements
+
+<div class="alert alert-warning" role="alert">
+<strong>Note:</strong> Jenkins on DCOS is only supported in version 1.4 and
+later. If you aren't yet running DCOS 1.4 and would like to try out Jenkins,
+please contact <a href="mailto:support@mesosphere.io">support@mesosphere.io</a>.
+</div>
+
+* A running Mesosphere DCOS cluster, version 1.4 or later
+  * Alternately, you can be running an open source Mesos cluster plus
+  Mesosphere's Marathon framework. If you're running Mesos open source, please
+  be sure that Docker Engine is configured and running on each of the agents,
```

Comment:
This might be a useful link to put in here: https://mesosphere.github.io/marathon/docs/native-docker.html